### PR TITLE
Saching/adding web pack

### PR DESCRIPTION
**Pull request by sachingupta**
This change will add module build support in test app still keeping output of webpack to public/app.js used by azure to deploy the code.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+  resolve: {
+    extensions: [ '.tsx', '.ts', '.js' ]
+  },
+  output: {
+    filename: 'app.js',
+    path: path.resolve(__dirname, 'public')
+  },
+  externals : {
+    microsoftTeams: '@microsoft/teams-js'
```
Comment (ydogandjiev) [inline webpack.config.js]: Why is this needed? Can't you just import microsoftTeams wherever it's needed?

Comment (sachingupta) [reply webpack.config.js]: to make it not be part of final outPut JS as i am planning to reference teams js separately
